client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -153,24 +166,22 @@ export default React.createClass( {
 	renderPlaceholder() {
 		return (
 			<div className="plugin-information is-placeholder">
-				<SectionHeader text={ this.translate( 'Plugin Information' ) } />
-				<Card>
 					<div className="plugin-information__wrapper">
-						<div className="plugin-information__version-info">
-							{ this.renderLastUpdated() }
-							<PluginVersion plugin={ this.props.plugin } />
-							{ this.renderLimits() }
-						</div>
 						<div className="plugin-information__rating-info">
-							{ this.renderStarRating() }
-							{ this.renderRatingsDetail() }
+							<PluginRatings
+								rating={ this.props.plugin.rating }
```

Comment:
Yeah I still have to give the placeholders a lot more love.
I just wanted to get the review process started. :)